selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/move/MoveItemsTest.java

```diff
@@ -87,6 +89,8 @@ public void closeForm() {
     try {
       if (refactor.isWidgetOpened()) {
         refactor.clickCancelButtonRefactorForm();
+      }
+      if (editor.isAnyTabsOpened()) {
```

Comment:
Please, add empty line before second `if` to improve readability.

Reply:
done